DICOMPlugins/DICOMM3DPlugin.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import absolute_import
+import glob
+import os
+import json
+import vtk
+import io
+import shutil
+import vtkSegmentationCorePython as vtkSegmentationCore
+import logging
+
+from base.DICOMM3DPluginBase import DICOMM3DPluginBase
+
+import slicer
+from DICOMLib import DICOMLoadable
+
+from SlicerDevelopmentToolboxUtils.mixins import ModuleLogicMixin
+from SlicerDevelopmentToolboxUtils.constants import DICOMTAGS
+from six.moves import range
+
+
+#
+# This is the plugin to handle translation of DICOM SEG objects
+#
+class DICOMM3DPluginClass(DICOMM3DPluginBase):
+
+  def __init__(self):
+    super(DICOMM3DPluginClass,self).__init__()
+    self.loadType = "DICOMM3D"
+
+  def examineFiles(self,files):
+    """ Returns a list of DICOMLoadable instances
+    corresponding to ways of interpreting the
+    files parameter.
+    """
+    loadables = []
+
+    # just read the modality type; need to go to reporting logic, since DCMTK
+    #   is not wrapped ...
+
+    for cFile in files:
```

Comment:
Style nitpick: we avoid using single letters to differentiate variable names.  Here, what does the 'c' mean?  Maybe you mean `candidateFile`?  If so that would be a good name.  With modern editors and auto completion this shouldn't make programming any harder but it makes reading the code easier.